napari/utils/events/event.py

```diff
@@ -98,13 +99,16 @@ class Event:
         All extra keyword arguments become attributes of the event object.
     """
 
-    def __init__(self, type: str, native: Any = None, **kwargs: Any) -> None:
+    @rename_argument("type", "type_name", "0.6.0")
+    def __init__(
+        self, type_name: str, native: Any = None, **kwargs: Any
+    ) -> None:
```

Comment:
This is basically vendored from vispy; are we sure all the interactions with vispy code are working as intended?

Reply:
Hm. Valid point. Tests are passing... 
I do not like overwriting python builtins. 

Has anyone a strong opinion?